rfcs/034-location_location_location/README.md

```diff
@@ -0,0 +1,403 @@
+
+    <table>
+      <tr><th>New LocationType</th><th>Sierra location names that match:</th></tr>
+      <tr>
+        <td>ClosedStores</td>
+        <td>
+          archives & mss well.coll<br/>
+          by appointment<br/>
+          closed stores<br/>
+          early printed books<br/>
+          iconographic collection<br/>
+          offsite<br/>
+          unrequestable
+        </td>
+      </tr>
+      <tr>
+        <td>OpenShelves</td>
+        <td>
+          biographies<br/>
+          folios<br/>
+          history of medicine<br/>
+          journals<br/>
+          medical collection<br/>
+          medicine & society collection<br/>
+          open shelves<br/>
+          quick ref collection<br/>
+          quick ref. collection<br/>
+          rare materials room<br/>
+          student coll
+        </td>
+      </tr>
+      <tr>
+        <td>OnExhibition</td>
+        <td>on exhibition</td>
+      </tr>
+      <tr>
+        <td>Conservation</td>
+        <td>conservation</td>
+      </tr>
+    </table>
+
+    This heuristic allows us to create locations in our controlled type for **94.7% of all items**.
+
+    Some Sierra items have a location like "as above" or "bound in above".
+    For these, we could apply an additional heuristic: if every other item has the same location (e.g. every other item is ClosedStores), then we give this item the same location.
+    Otherwise, we skip adding a Location to that item until we can come back and write more sophisticated logic (e.g. matching on shelfmark).
+
+    This additional heuristic would allow us to create locations in our controlled type for **99.97% of all items**.
+
+    The unmapped Sierra locations are as follows:
+
+    <table>
+      <tr><th>Sierra location</th><th># of items</th></tr>
+      <tr><td>bwith / bound in above</td><td>218</td></tr>
+      <tr><td>cwith / Contained in above</td><td>28</td></tr>
+      <tr><td>digit / Digitised Collections</td><td>10</td></tr>
+      <tr><td>wqrfe / Enquiry Line</td><td>2</td></tr>
+      <tr><td>temp1 / At Digitisation</td><td>1</td></tr>
+      <tr><td>wghig / Gallery</td><td>1</td></tr>
+      <tr><td>sgmip / sgmip</td><td>1</td></tr>
+      <tr><td>gblip / 215 Information Point</td><td>1</td></tr>
+    </table>
+
+    </details>
+
+    <details>
+    <summary>Proposed heuristic for matching Sierra holdings locations to controlled types</summary>
+
+    For Sierra holdings, the API only returns a location *code* (e.g. stax) via fixed field 40.
+
+    We will look up the location code to find the location name, then apply a similar rule to that which we apply for items.
+    If the location code or name are unrecognised, we will skip adding a location to the holding.
+
+    This heuristic allows us to create locations in our controlled type for **99.87% of all holdings**.
+    </details>
+
+*   We will add optional `"shelfLocation"` and `"shelfmark"` fields to the PhysicalLocation model.
```

Comment:
I have never known what to call the open shelves locations. It's currently labelled as _Location_ and the site talks about locations within the library. But `PhysicalLocation` having a `location` property is a bit weird. It's not necessarily on a shelf, of course. But everything does have a shelfmark...